hikari/impl/bot.py

```diff
@@ -870,7 +865,7 @@ async def _start_one_shard(
         end = date.monotonic()
 
         if new_shard.is_alive:
-            _LOGGER.info("Shard %s started successfully in %.1fms", shard_id, (end - start) * 1_000)
+            _LOGGER.debug("Shard %s started successfully in %.1fms", shard_id, (end - start) * 1_000)
```

Comment:
I think should stay in `info`, provides more feedback to the user

Reply:
theres already a gateway log to say it is ready, which immediately triggers this